keps/sig-cluster-lifecycle/kubeadm/0029-20180918-kubeadm-phases-beta.md

```diff
@@ -74,7 +74,8 @@ duplication of code or in some case inconsistencies between the init and phase i
 - This proposal doesn't include a plan for implementation of phases in workflows
   different than the `kubeadm init` workflow; such plans will be defined by the sig 
   during release planning for each cycle, and then documented in this KEP.
-  - v1.14 implementation of phases in the `kubeadm join` workflow (planned)
+  - v1.14 implementation of phases in the `kubeadm join` workflow
+  - v1.17 implementation of phases in the `kubeadm upgrade apply/plan/node` workflow
```

Comment:
Maybe just `kubeadm upgrade` [1]

Reply:
@yagonobre 👍 I did it this way because really it is the sub commands... but not all of them because I was looking at `diff` and am not sure it would apply... But if everyone is fine with just `kubeadm upgrade` that sounds good to me.

Reply:
Make senses, I don't have a hard opinion, at least from me it's up to you.

Reply:
I'll wait and see if others say anything. I too don't have a hard opinion :)

Reply:
Let's narrow it down to `kubeadm upgrade apply` for 1.17. `node` already has phases and `plan` can be seen as a sub-set of `apply` (they go together).